test/unit/org/apache/cassandra/schema/CreateTableValidationTest.java

```diff
@@ -48,4 +51,60 @@ public void testInvalidBloomFilterFPRatio() throws Throwable
         // sanity check
         createTable("CREATE TABLE %s (a int PRIMARY KEY, b int) WITH bloom_filter_fp_chance = 0.1");
     }
+
+    @Test
+    public void testCreateTableOnSelectedClusterColumn()
+    {
+        createTable("CREATE TABLE %s (pk int, ck1 int, ck2 int, v int, PRIMARY KEY ((pk),ck1, ck2)) WITH CLUSTERING ORDER BY (ck1 ASC);");
+    }
+
+    @Test
+    public void testCreateTableOnAllClusterColumns()
```

Comment:
```suggestion
    public void testCreateTableOnAllClusterColumns()
```
to be corrected on commit